test/dba-xq.xml

```diff
@@ -1,6 +1,6 @@
 <query xmlns='http://exist.sourceforge.net/NS/exist' start='1' max='99' wrap='no'>
 <text><![CDATA[
 xquery version '3.1';
-sm:is-dba(xmldb:get-current-user())
+sm:is-dba("admin")
```

Comment:
Do you really mean to code "admin" in here? You can get the current user via an XPath on `sm:id()`

Reply:
the xpath was throwing a hissyfit, the whole point of the test is just to see if with the newcredentials the query is executed or gets a `not-authenticated` error, so `admin` is fine, its not like we hide dba username anywhere else in the test file.